### PR TITLE
remove requestedAt param from retry call

### PR DESCRIPTION
**Pull request by docmerlin**
Remove requestedAt param from the retry api call
closes #1589

### CODE REVIEW

**Comment by docmerlin** [comment]
@mark-rushakoff Ok, I extended it to no longer accept requestedfAt and made a paired PR for idpe.

File: .circleci/config.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
 
   gotest:
     docker:
-      - image: circleci/golang:1.11
+      - image: circleci/golang:1.11.2
```
Comment (mark-rushakoff) [inline .circleci/config.yml]: Not necessary.

File: task/servicetest/servicetest.go [hunk 2]
```diff
@@ -430,7 +430,8 @@ func testTaskRuns(t *testing.T, sys *System) {
 
 		found := false
 		for _, mr := range meta.ManualRuns {
-			if mr.Start == mr.End && mr.Start == rc.Created.Now && mr.RequestedAt == requestedAtUnix {
+			if mr.Start == mr.End && mr.Start == rc.Created.Now { //&& (mr.RequestedAt ==requestedAtUnix||mr.RequestedAt == requestedAtUnix+1) {
+				fmt.Println(mr.RequestedAt, requestedAtUnix)
```
Comment (mark-rushakoff) [inline task/servicetest/servicetest.go]: Remove print.

File: task/servicetest/servicetest.go [hunk 3]
```diff
@@ -430,7 +430,8 @@ func testTaskRuns(t *testing.T, sys *System) {
 
 		found := false
 		for _, mr := range meta.ManualRuns {
-			if mr.Start == mr.End && mr.Start == rc.Created.Now && mr.RequestedAt == requestedAtUnix {
+			if mr.Start == mr.End && mr.Start == rc.Created.Now { //&& (mr.RequestedAt ==requestedAtUnix||mr.RequestedAt == requestedAtUnix+1) {
```
Comment (mark-rushakoff) [inline task/servicetest/servicetest.go]: Remove stray comment.

**Pull request merged**